src/api/fetchData.ts

```diff
@@ -1,9 +1,31 @@
-export const fetchData = async (url, query = {}, method = 'GET', body = null, headers = {}) => {
+interface FetchDataOptions {
+  url: string;
+  query?: Record<string, any>;
+  method?: string;
+  body?: any;
```

Comment:
P1:
any보다 더 적절한 타입을 쓰면 좋겠네요~
```suggestion
  body?: BodyInit;
```